tap_xero/client.py

```diff
@@ -131,6 +159,24 @@ def update_config_file(config, config_path):
     with open(config_path, 'w') as config_file:
         json.dump(config, config_file, indent=2)
 
+def is_not_status_code_fn(status_code):
+    def gen_fn(exc):
+        if getattr(exc, 'response', None) and getattr(exc.response, 'status_code', None) and exc.response.status_code not in status_code:
+            return True
+        # Retry other errors up to the max
+        return False
+    return gen_fn
+
+def retry_after_wait_gen():
+    while True:
+        # This is called in an except block so we can retrieve the exception
+        # and check it.
+        exc_info = sys.exc_info()
+        resp = exc_info[1].response
+        sleep_time_str = resp.headers.get('Retry-After')
+        LOGGER.info("Received 429 -- sleeping for %s seconds", sleep_time_str)
```

Comment:
@savan-chovatiya Can you change this message as:
"API rate limit exceeded -- sleeping for %s seconds"

Reply:
@karanpanchal-crest I have changed the message as per suggestion.